test/integration/web_console_extensions_test.go

```diff
@@ -1,265 +0,0 @@
-package integration
-
-import (
-	"bytes"
-	"crypto/tls"
-	"fmt"
-	"io/ioutil"
-	"net/http"
-	"net/url"
-	"os"
-	"path"
-	"path/filepath"
-	"strings"
-	"testing"
-
-	knet "k8s.io/apimachinery/pkg/util/net"
-
-	configapi "github.com/openshift/origin/pkg/cmd/server/api"
-	testserver "github.com/openshift/origin/test/util/server"
-)
-
-func TestWebConsoleExtensions(t *testing.T) {
```

Comment:
@spadgett you need to be sure you're testing this in the webconsole-server repo.  Do you think you can port it yourself or do you need help?

Reply:
These tests need to be reworked anyway since we're changing how extensions load. I can look at it in web-console-server.